docs/docs/keymaps/input-processors/scaler.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Scaler Input Processor
+sidebar_label: Scaler
+---
+
+## Overview
+
+The scaler input processor is used to scale the value of an input event that has a code matching the codes set on the scaler. Events with other codes will be ignored. Values are scaled by multiplying by the multiplier parameter, and then dividing by the divisor parameter.
+
+## Usage
+
+When used, a scaler takes two parameters, a multiplier and a divisor, e.g.:
```

Comment:
```suggestion
When used, a scaler takes two parameters that are positive integers, a multiplier and a divisor, e.g.:
```
I think this is true, and should distinguish from transformer's invert functionality as well.